Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM heroku/cedar:14
+
+RUN useradd -d /app -m app
+USER app
+WORKDIR /app/src
+
+ENV HOME /app
+ENV PORT 3000
+ENV PATH /app/.heroku/python/bin:/tmp/python-pack/bin:$PATH
+ENV STACK cedar-14
+ENV PYTHONHOME /app/.heroku/python
+ENV PYTHONPATH /app/
+ENV DOCKER_BUILD 1
+
+
+RUN mkdir -p /app/.heroku
+RUN mkdir -p /tmp/app
+RUN mkdir -p /app/src
+RUN mkdir -p /app/.profile.d
+RUN mkdir -p /tmp/python-pack
+RUN mkdir -p /tmp/cache
+RUN mkdir -p /tmp/environment
+
+
+WORKDIR /app/src
+WORKDIR /tmp/python-buildpack/bin
+
+WORKDIR /app/
+ONBUILD COPY . /app/
```

Comment:
I don't think you need to use `ONBUILD` for these unless there is another image you are creating that will derive from this.  For example, the python images have `ONBUILD` instructions that install `requirements.txt`  so that when you use `FROM python` your image will automatically install it without explicitly running the `pip install -r requirements.txt` command.